src/components/Post/Comment/CommentsContainerCard.tsx

```
@@ -0,0 +1,564 @@
+				) : null}
+			</div>
+
+			<div className=''>
+				<div className={`col-start-1 ${timelines.length >= 1 && 'xl:col-start-3'} col-end-13 mt-0`}>
+					{!!allComments?.length && !loading && (
+						<>
+							<Comments
+								disableEdit={isGrantClosed}
+								comments={allComments}
+							/>
+						</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)